src/modules/get-meals.js

```diff
@@ -0,0 +1,48 @@
+import Love from './love.png';
+import countItems from './items-count.js';
+
+const url = 'https://www.themealdb.com/api/json/v1/1/search.php?f=b';
+const maindiv = document.querySelector('.meals');
+const mealsNum = document.querySelector('.meals-count');
+
+const viewData = (pageLikes) => {
+  fetch(url, { mode: 'cors' }, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      const mealStore = [];
+      let counter = 0;
+      data.meals.forEach((info) => {
+        if (counter < 6) {
+          mealStore.push(info);
+          counter += 1;
+        }
+      });
+      const mealCounter = countItems(mealStore);
+      mealsNum.innerHTML = `(${mealCounter})`;
+      mealStore.forEach((element) => {
+        const id = element.idMeal;
+        const mealLikes = pageLikes.find((like) => like.item_id === id);
+        const mealsDiv = document.createElement('div');
+        mealsDiv.classList.add('meals-div');
+        mealsDiv.innerHTML = `
+        <div class="meal-card">
+          <img class="meal-img" src=${element.strMealThumb}>
+          <div class="inner-part">
+            <p>${element.strMeal}</p>
+            <button  class="likes-button"><img id="${element.idMeal}" alt="likes" class="likes-icon" src="${Love}"></button>
+          </div>
+          <p id="likes-count-${element.idMeal}" class="likes-count" >${mealLikes ? mealLikes.likes : 0} Likes</p>
+          <button id= "${element.idMeal}" class="buttons" > Comments </button>
+        </div>
+        `;
+        maindiv.appendChild(mealsDiv);
+      });
+    });
+};
```

Comment:
- [Required] I think since this function fetches some data from an `API` online and it might take some time it would be better if you use `async` and `await` with the `fetch` method to receive the data from the API. The keyword `async` before a function makes the function return a promise and the `await` keyword makes the function pause the execution and wait for a resolved promise before it continues which in turn makes your function run synchronously without an error. Please see my first comment in the review on how you can make this function use `async` and `await`.